metal-support/cmd/metal-support.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	"github.com/openshift-metal3/dev-scripts/metal-releases/pkg/commands"
+	"github.com/urfave/cli/v2"
+)
+
+func main() {
+
+	app := &cli.App{
+		Name:                 "metal-support",
+		Usage:                "",
+		UsageText:            "",
+		Description:          "A tool for monitoring/troubleshooting metal-ipi OpenShift CI releases",
```

Comment:
metal-support describes the role, not really this tool - would it make sense to call it e.g `metal-ci-status`?

Relatedly, instead of a `metal-support` top-level directory, we could perhaps have `tools/metal-ci-status` ?

Reply:
The basic idea was to deploy a tool to help the metal-support role, a tool composed by different commands. In this first draft we added two completely different commands, one for detecting intermittent failures, and another one to display the metal jobs as a classic ci/cd wall monitor. Sounds fine to move it under `tools`, but `metal-ci-status` seems to fit partially the scope of the tool